modules/codechanges.adoc

```diff
@@ -55,7 +55,7 @@ Username: {{GITLAB_USER}}
 Password: {{GITLAB_PASSWORD}}
 ====
 
-Click the Settings link on the top right of the screen, and then click "Webhooks":
+Click the Settings link on the top right of the screen, and then click "Integrations":

```

Comment:
I can try to update the workshop provisioner to use these. Which template are you using?

I guess just merge it -- we can talk around it if it's a problem.